app/src/main/java/com/yeonae/chamelezone/view/mypage/mycourse/adapter/MyCourseRvAdapter.kt

```diff
@@ -51,7 +51,11 @@ class MyCourseRvAdapter(private var items : ArrayList<Course>) : RecyclerView.Ad
     class MyCourseViewHolder(parent: ViewGroup) : RecyclerView.ViewHolder(
         LayoutInflater.from(parent.context).inflate(R.layout.item_my_course, parent, false)
     ) {
-        fun bind(item: Course, clickListener: OnClickListener?, locationListener: GetLocationListener) {
+        fun bind(
+            item: Course,
+            clickListener: OnClickListener?,
```

Comment:
```suggestion
            clickListener: OnClickListener,
```
Nullable이 아니어도 될 것 같습니다.

Reply:
수정했습니당